src/streaming/MediaPlayerEvents.js

```diff
@@ -105,6 +105,10 @@ class MediaPlayerEvents extends EventsBase {
          * @event MediaPlayerEvents#PERIOD_SWITCH_STARTED
          */
         this.PERIOD_SWITCH_STARTED = 'streamswitchstarted';
+
+        this.QUALITY_CHANGE_START = 'qualityChangeStart';
```

Comment:
Are these events external? The convention for events that go to the client is to be lower case - it came in in 2.0 iirc.